samples/matter/light_bulb/Kconfig.sysbuild

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "${ZEPHYR_BASE}/share/sysbuild/Kconfig"
+
+config NRF_DEFAULT_MULTIPROTOCOL
```

Comment:
Is it going to work for the hci_rpmsg case (Wi-Fi) as well?

Reply:
No, as I don't see support for nRF70 family added. @nordicjm, can you share what the plan is?

Reply:
What is the question?

Reply:
If the nRF70 boards are currently supported or if there is a plan to add support for them soon.

Reply:
What does the nRF70 board need?

Reply:
`nrf7002dk_nrf5340_cpuapp` and `nrf7002dk_nrf5340_cpunet` targets. We need them to support all required Nordic platforms in Matter. I mean, this is not needed right now because this is an initial port. I want to know if there is a plan to support these any soon.

Reply:
This is gated on hwmv2 support upstream. If you want support for it before then, the board names can be added to the Kconfigs and cmake files in `sysbuild` and it will work like the nrf5340dk

Reply:
Is it just a matter of extending https://github.com/nrfconnect/sdk-nrf/blob/d97238ff9328533a7590e934b7680db949c0ad71/sysbuild/Kconfig.netcore#L7 and other board related bits in this file?

Reply:
that is part of it, the other part is the cmake parts in the same folder